src/Microsoft.Health.Fhir.Stu3.Core.UnitTests/Features/Export/ExportAnonymizerFactoryTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.Health.Fhir.Core.Features.Operations;
+using Microsoft.Health.Fhir.Core.Features.Operations.Export;
+using NSubstitute;
+using Xunit;
+
+namespace Microsoft.Health.Fhir.STU3.Core.UnitTests.Features.Export
+{
+    public class ExportAnonymizerFactoryTests
+    {
+        private const string SampleConfiguration =
+            @"
+{
+	""fhirPathRules"": [
+		{
+			""path"": ""Patient"",
+			""method"": ""redact""
+		}
+	]
+}";
+
+        private const string InvalidConfiguration =
+    @"
+{
+	""fhirPathRules"": [
+		{
+			""path"": ""invalid()"",
+			""method"": ""redact""
+		}
+	]
+}";
+
+        [Fact]
+        public async Task GivenAValidAnonymizationConfiguration_WhenCreate_AnonymizerShouldBeCreated()
```

Comment:
As with the other file Perhaps _WhenCreatingAnonymizer_

Reply:
Thanks, updated.